test/fixtures/terms.yml

```diff
@@ -2,16 +2,20 @@
 #
 # Table name: terms
 #
-#  id         :integer          not null, primary key
-#  phrase     :string
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
-#  flag       :boolean
+#  id             :integer          not null, primary key
+#  phrase         :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  flag           :boolean
+#  fingerprint_id :integer
```

Comment:
I can see this perspective. I made the last round of changes because I wanted to confirm that not populating the fingerprints as part of the migration wouldn't cause a problem, and it looks like that's the case (no code changes were needed, just test changes to adapt to the new starting point).

Let me see if I can find a comfortable test position here...